src/tespy/components/nodes.py

```diff
@@ -1415,6 +1415,37 @@ def additional_derivatives(self, increment_filter, k):
             j += 1
         k += 1
 
+    def exergy_balance(self, Tamb_val_SI, bus):
+        r"""
+        Calculate exergy balance of a merge.
+        
+        Parameters
+        ----------
+        Tamb_val_SI : float
+            Ambient temperature in K.
+        bus  : tespy.connections.bus
+            Energy flows in network. Used to calculate product exergy
+            or fuel exergy of turbines and pumps.
+            
+        Note
+        ----
+         .. math::
+
+                E_{P} = \sum_{n_{cold}=0}^N \dot{m}_{in,n_{cold}} \cdot (e_{ph,out} - e_{ph,in,n_{cold}})
+                E_{F} = \sum_{n_{hot}=0}^N \dot{m}_{in,n_{hot}} \cdot (e_{ph,out} - e_{ph,in,n_{hot}})
+        """
+        self.E_P = 0
+        for i in range(0, len(self.inl)):
+            if self.inl[i].T.val_SI <= self.outl[0].T.val_SI:
+                self.E_P += self.inl[i].m.val_SI * (self.outl[0].ex_physical 
+                                                    - self.inl[i].ex_physical)
+
+        self.E_F = 0
+        for i in range(0, len(self.inl)):
+            if self.inl[i].T.val_SI > self.outl[0].T.val_SI:
+                self.E_F += self.inl[i].m.val_SI * (self.inl[i].ex_physical 
+                                                    - self.outl[0].ex_physical)
+
```

Comment:
This might only work with pure fluids. When mixing different fuels the composition of the fluids at the inlets and the outlet will be different, and therefore the reference might not be correct anymore. We will need to check this.

Reply:
Oh yes, might not work. I forgot the case of mixing different fluids